Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -118,11 +116,50 @@ final class AutoResizingMaskViewSource {
     return rightFlexibleTopBottomView
   }()
 
-  private func bakeViewWithFrame(_ frame: CGRect) -> UIView {
-    let view = UIView(frame: frame)
+  private func bakeViewWithFrame(_ frame: CGRect) -> UserInterfaceStyleAwareView {
+    let view = UserInterfaceStyleAwareView(frame: frame)
     view.layer.cornerRadius = type(of: self).cornerRadius
     view.layer.borderWidth = type(of: self).borderWidth
-    view.layer.borderColor = type(of: self).borderColor
+    view.layer.borderColor = view.borderColor
     return view
   }
 }
+
+class UserInterfaceStyleAwareView: UIView {
+
+  var borderColor: CGColor {
```

Comment:
Can this be `private`? I wonder if you're not better off returning a `UIColor` from this and only doing the `.cgColor` when assigning it to the border. Potato vs. Potatoes 😉 

Don't remember the convention used on this project - but I'd advise moving private properties after public ones, and after a `// MARK: Private`